src/components/IconButtonBar/IconButtonBar.stories.js

```diff
@@ -35,11 +35,16 @@ import { TooltipDirection } from '../IconButton/IconButton';
 storiesOf(patterns('IconButtonBar'), module).add(
   'default',
   () => {
-    const size = select('size', sizes, sizes[0]);
+    const size = select('size', sizes, 'sm');
     /* eslint-disable no-nested-ternary */
     const actions = [
       {
         className,
+        divider: select(
```

Comment:
Would you mind including a `null` / `undefined` option to provide an example of the `IconButtonBar` without any dividers?